lib/visualization.js

```diff
@@ -273,6 +279,17 @@ class BayesGridVis extends GridVisualization {
 		this.weights = [];
 		this.color_normalization = this.M * this.N * 100;
 		this.ig_trace = trace.ig;
+		this.plan_trace = trace.plan_trace;
+
+		this.svg
+			.attr('height', this.height + this.d + 70); //TODO fix magic no.
```

Comment:
we should do a pass over the vis classes to clean out the magic numbers. i think there's a few more that we can maybe get rid of or put in configs.